.github/workflows/common.yml

```diff
@@ -0,0 +1,38 @@
+name: Common CI workflow
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
```

Comment:
i think it's better to restrict running this workflow only on this specific GitHub repo, as done in

https://github.com/kxcinc/bindoj/blob/88ea5748a5d7eaedbf52f5b22dc3fabf009ec6a1/.github/workflows/all-variants.yml#L35

we can encourage the template users to enable them in the README, which essentially makes it an opt-in "feature" of this repo, as its using kxcdev/ocaml-general-gha instead of more official setup-ocaml

Reply:
I have added it.